### PR TITLE
doc update for 2020.1 PU1

### PR DESCRIPTION
**Pull request by imrickysu**
- Split build and install documents; update installation pre-request commands. 
- Update system requirements to remove duplicated info with build.rst and install.rst
- Update system requirements to refer supported OS to release notes
- Update index to add install.rst

### CODE REVIEW

File: src/runtime_src/doc/toc/build.rst [hunk 1]
```diff
@@ -1,46 +1,62 @@
 .. _build.rst:
 
-Building and Installing Software Stack
+Building the XRT Software Stack
 --------------------------------------
```
Comment (sonals) [inline src/runtime_src/doc/toc/build.rst:5]: Please match the --- with the length of the heading

File: src/runtime_src/doc/toc/platforms.rst [hunk 2]
```diff
@@ -1,12 +1,11 @@
 .. _platforms.rst:
 
 
-Platform Overview
-*****************
+XRT and Vitis™ Platform Overview
+*************************
```
Comment (sonals) [inline src/runtime_src/doc/toc/platforms.rst:5]: Please match the *** with the length of the heading

File: src/runtime_src/doc/toc/system_requirements.rst [hunk 3]
```diff
@@ -13,29 +13,27 @@ Host Platform for PCIe Accelerator Cards
 Supported Xilinx® Accelerator Cards are listed in :ref:`platforms.rst`.
 
 
-Software Platform for PCIe Accelerator Cards
+XRT Software Stack for PCIe Accelerator Cards
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (sonals) [inline src/runtime_src/doc/toc/system_requirements.rst:17]: Please match the ~~~ with the length of the heading

File: src/runtime_src/doc/toc/system_requirements.rst [hunk 4]
```diff
@@ -13,29 +13,27 @@ Host Platform for PCIe Accelerator Cards
...
+To install XRT on the host, please refer to page :ref:`install.rst` for dependencies installation steps and XRT installation steps.
 
-To build a custom
-version of XRT, please follow the instructions in :ref:`build.rst`.
+To build a custom XRT package, please refer to page :ref:`build.rst` for dependencies installation steps and building steps.
 
-
-MPSoC Based Embedded Platforms
+XRT Software Stack for Embedded Platforms
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (sonals) [inline src/runtime_src/doc/toc/system_requirements.rst:31]: Please match the ~~~ with the length of the heading

**Comment by imrickysu** [comment]
Title underline length updated. Please review again.

**Review by chvamshi-xilinx — APPROVED** [review]

**Comment by chvamshi-xilinx** [comment]
Looks like Travis build stalled. Retesting as this is for PU1

**Comment by chvamshi-xilinx** [comment]
retest this please

**Comment by xilinxgitops** [comment]
retest this please

**Comment by imrickysu** [comment]
@sonals  @stsoe  could you review and approve?

**Pull request merged**